### ISSUE TITLE
Frontmatter check warns about commas in description

### ISSUE THREAD

**Issue by infotexture** [agent-ecosystem/skill-validator#71]
## What happened?

The frontmatter check says the description passes, but warns that the `description` field reads like a comma-separated keyword list rather than prose. 

## What did you expect?

The description is genuinely prose; the validator seems to be just counting commas. Since this is a skill describing UI element types, commas are unavoidable without making the description awkward.

## How to reproduce

Sample description field:

```yaml
description: Helps agents write and edit interface copy (microcopy) for
  digital products — buttons, labels, error messages, forms, onboarding
  flows, empty states, and help text. Use this skill whenever you need to
  produce or improve any text that appears in an app, website, or software
  UI. It applies four core quality standards (purposeful, concise,
  conversational, and clear) and ships with accessibility guidelines,
  research-backed readability benchmarks, error-message patterns, tone
  adaptation frameworks, and fillable templates.
```

Resulting warning:

```bash
  ⚠ description has 8 comma-separated segments, most very short —
    this looks like a keyword list; per the spec, the description should
    concisely describe what the skill does and when to use it
```

## Environment

- skill-validator version: 1.5.4

**Comment by infotexture** [comment]
P.S. Thanks so much for building this, it's incredibly useful! 🙏 🙇 

— Not filing this to complain, just wanted to provide feedback in the spirit of improvement. 🙂

**Comment by dacharyc** [comment]
Thanks for the bug report, and I'm glad you're finding the tool useful! The keyword stuffing heuristic has been through a few iterations so far; I've just added a new prose density gate to try to detect keyword stuffing more accurately. I appreciate the info here to help me refine.

I've just released v1.5.5 with the updates to the keyword stuffing heuristic; I used your description in the new test, and it's passing, so hopefully this solves your problem. 🤞

**Comment by infotexture** [comment]
It does indeed, thanks for the quick fix in #73!  🥇 

I'll provide more feedback in a separate issue, closing this as resolved in [v1.5.5](https://github.com/agent-ecosystem/skill-validator/releases/tag/v1.5.5).
